<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student</class>
 <widget class="QWidget" name="student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnSort">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>排序</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>801</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>540</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>查询学号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="textFindnum"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnRef">
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnChange">
      <property name="text">
       <string>增改信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDel">
      <property name="text">
       <string>删除信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>62</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>学号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>性别</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>年龄</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>班级</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>姓名</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkDesc">
      <property name="text">
       <string>降序</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>189</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
